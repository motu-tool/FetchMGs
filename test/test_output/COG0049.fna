version https://git-lfs.github.com/spec/v1
oid sha256:845c0435e09b316840e3656947b8c9ac2408c33f1ebfbbb9c4fc5919a07b586e
size 2975
