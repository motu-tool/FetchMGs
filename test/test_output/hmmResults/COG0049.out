# hmmsearch :: search profile(s) against a sequence database
# HMMER 3.3.2 (Nov 2020); http://hmmer.org/
# Copyright (C) 2020 Howard Hughes Medical Institute.
# Freely distributed under the BSD open source license.
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
# query HMM file:                  /nfs/home/fieldc/.local/lib/python3.7/site-packages/fetchmgs/data/COG0049.hmm
# target sequence database:        example_input/example_data_genomes.faa
# output directed to file:         example_output/hmmResults/COG0049.out
# per-dom hits tabular output:     example_output/hmmResults/COG0049.dom
# show alignments in output:       no
# max ASCII text line length:      unlimited
# sequence reporting threshold:    score >= 120
# number of worker threads:        1
# - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Query:       COG0049  [M=163]
Scores for complete sequences (score includes all domains):
   --- full sequence ---   --- best 1 domain ---    -#dom-
    E-value  score  bias    E-value  score  bias    exp  N  Sequence           Description
    ------- ------ -----    ------- ------ -----   ---- --  --------           -----------
    1.2e-73  246.6   3.0    1.3e-73  246.5   3.0    1.0  1  1089456.NCGM2_5512  protein_id="BAK92333.1" gene="rpsG" product="30S ribosomal protein S7" GI="348036973" note="similar to GI:116052302; similar to PA14_08810 of P. aeruginosa UCBPP-PA14" project_id="73453"
    6.9e-70  234.3   0.8    7.7e-70  234.2   0.8    1.0  1  1148.SYNGTS_1748    protein_id="BAK50496.1" gene="rps7" product="30S ribosomal protein S7" GI="339274009" note="ORF_ID:sll1097" project_id="67081"
    1.5e-69  233.2   4.7    1.7e-69  233.1   4.7    1.0  1  944547.ABLL_2420    protein_id="BAK74295.1" product="30S ribosomal protein S7" GI="345472845" project_id="61721"
    6.6e-69  231.2   4.4    7.3e-69  231.0   4.4    1.0  1  944546.ABED_1797    protein_id="BAK71514.1" product="30S ribosomal protein S7" GI="345470063" project_id="61719"
    9.8e-65  217.6   1.7    1.1e-64  217.5   1.7    1.0  1  1037409.BJ6T_43830  protein_id="BAL09652.1" gene="rpsG" product="30S ribosomal protein S7" GI="354956973" project_id="67463"


Domain annotation for each sequence:
>> 1089456.NCGM2_5512  protein_id="BAK92333.1" gene="rpsG" product="30S ribosomal protein S7" GI="348036973" note="similar to GI:116052302; similar to PA14_08810 of P. aeruginosa UCBPP-PA14" project_id="73453"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  246.5   3.0   2.8e-77   1.3e-73       2     157 ..       1     156 []       1     156 [] 1.00

>> 1148.SYNGTS_1748  protein_id="BAK50496.1" gene="rps7" product="30S ribosomal protein S7" GI="339274009" note="ORF_ID:sll1097" project_id="67081"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  234.2   0.8   1.7e-73   7.7e-70       2     157 ..       1     156 []       1     156 [] 1.00

>> 944547.ABLL_2420  protein_id="BAK74295.1" product="30S ribosomal protein S7" GI="345472845" project_id="61721"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  233.1   4.7   3.6e-73   1.7e-69       4     157 ..       2     155 .]       1     155 [] 0.99

>> 944546.ABED_1797  protein_id="BAK71514.1" product="30S ribosomal protein S7" GI="345470063" project_id="61719"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  231.0   4.4   1.6e-72   7.3e-69       4     157 ..       2     155 .]       1     155 [] 0.99

>> 1037409.BJ6T_43830  protein_id="BAL09652.1" gene="rpsG" product="30S ribosomal protein S7" GI="354956973" project_id="67463"
   #    score  bias  c-Evalue  i-Evalue hmmfrom  hmm to    alifrom  ali to    envfrom  env to     acc
 ---   ------ ----- --------- --------- ------- -------    ------- -------    ------- -------    ----
   1 !  217.5   1.7   2.3e-68   1.1e-64      28     157 ..       1     130 []       1     130 [] 1.00



Internal pipeline statistics summary:
-------------------------------------
Query model(s):                            1  (163 nodes)
Target sequences:                      23272  (7230323 residues searched)
Passed MSV filter:                       616  (0.0264696); expected 465.4 (0.02)
Passed bias filter:                      465  (0.0199811); expected 465.4 (0.02)
Passed Vit filter:                        38  (0.00163286); expected 23.3 (0.001)
Passed Fwd filter:                         5  (0.00021485); expected 0.2 (1e-05)
Initial search space (Z):              23272  [actual number of targets]
Domain search space  (domZ):               5  [number of targets reported over threshold]
# CPU time: 0.12u 0.00s 00:00:00.12 Elapsed: 00:00:00.07
# Mc/sec: 15717.30
//
[ok]
